starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,84 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+h2{
+  margin-left: 15%;
+}
+
+ul {
+ counter-reset: custom;
+ list-style-type: none;
+ line-height: 1.6;
+ margin: 0 auto;
+}
+
+ul li:before {
+ content: '- ';
+}
+
+ol {
+ counter-reset: custom;
+ list-style-type: none;
+ line-height: 1.6;
+ margin: 0 auto;
+}
+
+ol li:before {
+ content: counter(custom)') ';
+ counter-increment: custom;
+}
+
+
+.french-onion-image h1{
+  color: white;
+  font-size: 3em;
+  font-weight: bold;
+  text-align: center;
+  padding: 20%;
+}
+
+.french-onion-image{
+  background-position: center;
+  background-image: url("../images/soup.jpg");
+  background-repeat: no-repeat;
+  background-size: 65%;
+}
+
+.anectdote{
+  text-align:center;
+  margin-left: 15%;
+  margin-right: 15%;
+}
+
+.recipe-info{
+  background-position: center;
+  background-image: url("../images/recipe-info.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  width: 550px;
+  height: 120px;
+  margin-left: 560px;
+}
+
+.section-dividers{
+  width: 75%;
+}
+
+.ingredient-list{
+  margin-left: 18%;
+}
+
+.cooking-info{
+  background-position: center;
+  background-image: url("../images/cooking-info.png");
+  background-repeat: no-repeat;
+  background-size: 75%;
+  width: 550px;
+  height: 120px;
+  margin-left: 560px;
+}
+
+.directions-list{
+  margin-left: 18%;
```

Comment:
you should add margin or padding to both left and right in order to position everything more in the center like the site you were cloning.